recipes/safe-pysha3/conda_build_config.yaml

```diff
@@ -0,0 +1,17 @@
+python:
```

Comment:
This file is not needed and should be removed.

Reply:
Thank you for the review. This package cannot be used with python 3.8